test/jdk/java/awt/Focus/UnaccessibleChoice/AccessibleChoiceTest.java

```diff
@@ -66,44 +65,42 @@ private void test() throws IOException {
             init();
             start();
         } finally {
-            if (frame != null) frame.dispose();
-            if (win != null) win.dispose();
+            if (frame != null) {
+                frame.dispose();
+            }
         }
     }
 
     public void init() {
-        win.setLayout (new FlowLayout ());
-        win.add(def);
+        frame.setLayout(new FlowLayout());
+        frame.add(def);
         def.addFocusListener(new FocusAdapter() {
                 public void focusGained(FocusEvent e) {
                     go.countDown();
                 }
             });
         choice.add("One");
         choice.add("Two");
-        win.add(choice);
-    }
-
-    public void start () throws IOException {
+        frame.add(choice);
+        frame.pack();
+        frame.setLocationRelativeTo(null);
         frame.setVisible(true);
-        win.pack();
-        win.setLocation(100, 200);
-        win.setVisible(true);
+    }
 
-        Robot robot = null;
+    public void start() throws IOException {
+        Robot robot;
         try {
             robot = new Robot();
         } catch (Exception ex) {
             throw new RuntimeException("Can't create robot");
         }
         robot.waitForIdle();
         robot.delay(1000);
-        robot.setAutoDelay(150);
         robot.setAutoWaitForIdle(true);
 
         // Focus default button and wait till it gets focus
         Point loc = def.getLocationOnScreen();
-        robot.mouseMove(loc.x+2, loc.y+2);
+        robot.mouseMove(loc.x + 5, loc.y + 5);
```

Comment:
For test stability it is better move the click to the center of the button.

robot.mouseMove(loc.x + def.getWidth() / 2, loc.y + def.getHeight() / 2);

Reply:
Sure. Modified.